pyproject.toml

```diff
@@ -51,6 +51,7 @@ test = [
     'ci-watson >=0.5.0',
     'pytest >=4.6.0',
     'pytest-astropy',
+    'metrics_logger @ git+https://github.com/spacetelescope/metrics_logger.git',
```

Comment:
A direct URL dependency like this will prevent this package from being published on PyPi. See the the Attention note here: https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#direct-url-dependencies

Please update this to a PyPi reference.